src/EventStore.Core/Util/DefaultFiles.cs

```diff
@@ -5,9 +5,8 @@
 
 namespace EventStore.Core.Util;
 
-
 public static class DefaultFiles {
     public static readonly string DefaultConfigFile = RuntimeInformation.IsWindows
         ? string.Empty
-        : "eventstore.conf";
+        : "kurrentdb.conf";
```

Comment:
Shouldn't we have some grace period with that file? It breaks basically our `.deb` file and docker configuration in the wild.

What about issuing a deprecation notice when `eventstore.conf` is used instead? We could get rid completely of the `eventstore.conf` in the next release?

Reply:
@YoEight Yeah, I'll add support for still using the `eventstore.conf` file if `kurrentdb.conf` doesn't exist

Do you think we should provide the same grace period for the default locations such as `/etc/eventstore` and `/var/lib/eventstore` as well? That's trickier, but should still be doable